Tests/CADCacheAdvanceTests/CADCacheAdvanceTests.m

```diff
@@ -0,0 +1,107 @@
+//
+//  Created by Dan Federman on 4/24/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#if GENERATED_XCODE_PROJECT
+// These imports work when working within the generated CacheAdvance.xcodeproj used in CI.
+#import <CADCacheAdvance/CADCacheAdvance-Swift.h>
+#import <XCTest/XCTest.h>
+#else
+// These imports work when working within Package.swift.
+@import CADCacheAdvance;
+@import XCTest;
+#endif
+
+/// Tests that exercise the API of CADCacheAdvance.
+/// - Note: Since CADCacheAdvance is a thin wrapper on CacheAdvance<Data>, these tests are intended to do nothing more than exercise the API.
+@interface CADCacheAdvanceTests : XCTestCase
+@end
+
+@implementation CADCacheAdvanceTests
+
+- (void)test_fileURL_returnsUnderlyingFileURL;
```

Comment:
I don't think we need these trailing semicolons since we're defining the method, not declaring it.

Reply:
We don't need them. It's a holdover from my time at Square. Square's Objective-C style guide had every method having a `;` at the end of the line, even in implementation files. This was to aid in the copy/pasting method signatures between .m and .h files. It stuck with me 🙂